db/migrate/20231107210042_create_ingredients.rb

```diff
@@ -0,0 +1,11 @@
+class CreateIngredients < ActiveRecord::Migration[7.0]
+  def change
+    create_table :ingredients do |t|
+      t.string :name
+      t.integer :calories
+      t.references :dish, null: false, foreign_key: true
```

Comment:
Ingredients should *not* have a foreign key to dish here. With the many to many relationship, the foreign keys live in the joins table only. Ingredients don't belong to dishes, but rather they can have many dishes and dishes can have many ingredients